frontend/packages/dev-console/src/components/import/app/ApplicationSelector.tsx

```diff
@@ -46,7 +46,7 @@ const ApplicationSelector: React.FC<ApplicationSelectorProps> = ({
     const noApplicationsAvailable = _.isEmpty(applicationList);
     setApplicationsAvailable(!noApplicationsAvailable);
     if (noApplicationsAvailable) {
-      setFieldValue('application.selectedKey', CREATE_APPLICATION_KEY);
+      setFieldValue('application.selectedKey', '');
```

Comment:
I assume this change was needed because CREATE_APPLICATION_KEY is tied to showing the dropdown?

Reply:
correct.